src/literature/components/TabnameWithCount/TabnameWithCount.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Icon } from 'antd';
+
+import './TabnameWithCount.scss';
+
+class TabnameWithCount extends Component {
+  render() {
+    const { name, loading, count } = this.props;
+    return (
+      <span className="__TabnameWithCount__">
+        <span>{name}</span>
+        <span className="loading-or-count">
```

Comment:
``ml1`` is not enough?

Reply:
currently is `0.5em`, I think `1em` could look too much.

Reply:
can we check?